bftengine/src/preprocessor/RequestProcessingState.cpp

```diff
@@ -257,15 +264,17 @@ PreProcessingResult RequestProcessingState::definePreProcessingConsensusResult()
                  << KVLOG(batchCid_, reqSeqNum_, reqCid_, numOfReceivedReplies_, numOfRequiredEqualReplies_));
     return CONTINUE;
   }
-  if (primaryPreProcessResultLen_ == 0) {
```

Comment:
What is a case in which we can have **primaryPreProcessResultLen_ == 0**. I understand with the modification we consider **preprocessingRightNow_** and don't proceed with the logic that follows in the function, but I was wondering how can **primaryPreProcessResultLen_ == 0**. If I understand correctly we now ignore this case?

Reply:
We can verify primary pre-processing completion either through primaryPreProcessResultLen_ != 0 or preprocessingRightNow_ = false. The second one looks more native.